rest/audit_test.go

```diff
@@ -974,6 +1120,22 @@ func requireDocumentReadEvents(rt *RestTester, output []byte, docID string, docV
 	require.Equal(rt.TB(), docVersions, docVersionsFound)
 }
 
+// requireAttachmentEvents validates that create attachment events
```

Comment:
Is the idea that this expects that eventID is one of the attachment crud event IDs?